nl.g4d.Girens.json

```diff
@@ -0,0 +1,288 @@
+                            "sha256" : "881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2"
+                        }
+                    ],
+                    "modules" : [
+                        {
+                            "name" : "fribidi",
+                            "cleanup" : [
+                                "/bin",
+                                "/include",
+                                "/lib/pkgconfig",
+                                "/lib/*.la",
+                                "/share/man"
+                            ],
+                            "sources" : [
+                                {
+                                    "type" : "archive",
+                                    "url" : "https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2",
+                                    "sha256" : "6a64f2a687f5c4f203a46fa659f43dd43d1f8b845df8d723107e8a7e6158e4ce"
+                                }
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name" : "ffmpeg",
+                    "cleanup" : [
+                        "/include",
+                        "/lib/pkgconfig",
+                        "/share/ffmpeg/examples"
+                    ],
+                    "config-opts" : [
+                        "--enable-shared",
+                        "--disable-static",
+                        "--enable-gnutls",
+                        "--disable-doc",
+                        "--disable-ffplay",
+                        "--enable-version3"
+                    ],
+                    "sources" : [
+                        {
+                            "type" : "git",
+                            "url" : "https://github.com/FFmpeg/FFmpeg.git",
+                            "commit" : "6a8cc8696339f9c494da9aa54068c7dc07f75298"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "python-mpv",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "pip3 install --prefix=/app ./"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/jaseg/python-mpv",
+                    "commit" : "9b4faea03050b161328c4c83f67b407ad0b10c80"
+                }
+            ]
+        },
+    	{
+            "name" : "PlexAPI",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "pip3 install --prefix=/app ./"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/tijder/python-plexapi",
+                    "commit" : "abac9173f838c306497f3b69899af0b3ae3f0eb5"
+                }
+            ],
+            "modules" : [
```

Comment:
It will be easier to maintain these python deps using the generator here: https://github.com/flatpak/flatpak-builder-tools/tree/master/pip